.travis.yml

```diff
@@ -1,12 +1,13 @@
 language: node_js
 
-node_js:
-  - 4
+sudo: false
 
-before_install:
-  - npm config set spin false
+node_js:
+  - "6"
 
-script:
-  - npm test
```

Comment:
Is `script: npm test` not needed anymore?

Reply:
It was not needed in the past either when you specify `language: node_js` it's the default :)